Makefile

```diff
@@ -158,13 +158,21 @@ npm-refresh: ## Removes and rebuilds node app containers, images, volumes.
 	@make npm-down n=app-editor; make build n=app-editor; make up n=app-editor;
 
 ##############################################################################
-# Flyway Database Migration Commands
+# Database Commands
 ##############################################################################
 
 db-update: ## Run the flyway database migration update (requires maven to be installed).
 	@echo "$(P) Run the flyway database migration update"
 	@cd libs/java/dal/db; make db-update;
 
+db-refresh: ## Stop and delete the database container and volume, then rebuilds and starts.
```

Comment:
Makes it quicker to refresh the database.  Keycloak also uses this database and must be regenerated.